props/saber_BC_buttons.h

```diff
@@ -2042,6 +2064,154 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
 
 #endif  // BC_DUAL_BLADES
 
+  // Manual blade array selection
+  // even with Real TIme Blade ID active
+  bool manual_blade_array_active = false;
+  size_t last_real_best_config = NELEM(blades);
+  size_t real_best_config = NELEM(blades);
+  size_t fake_best_config = NELEM(blades);
+  size_t prev_real_best_config = NELEM(blades);
+
+#ifdef BLADE_ID_SCAN_MILLIS
+  // Must be called from loop()
```

Comment:
"must be called from loop" is the same comment as PollScanId() in prop base, but presumably this is not the same function, or you wouldn't have overridden it. Please update the comment to explain what this function does differently from the base function, otherwise the reader (me) will have to compare manually.

Reply:
"must be called from loop"  was from the initial copy . removed.
Comments added to hopefully explain the functions.